modules/service/src/main/resources/db/migration/V0369__itc_result.sql

```diff
@@ -0,0 +1,19 @@
+
+--
+-- ITC result table associated with a hash of ITC inputs.
+--
+CREATE TABLE t_itc_result (
+  c_program_id      d_program_id     NOT NULL,
+  c_observation_id  d_observation_id NOT NULL,
+  c_target_id       d_target_id      NOT NULL,
+  c_hash            text             NOT NULL,
+  c_exposure_time   interval         NOT NULL,
+  c_exposure_count  int4             NOT NULL,
+  c_signal_to_noise numeric(10,3)    NOT NULL,
+  c_version         text             NOT NULL,
+  c_data            text             NULL,
```

Comment:
Storing the ITC version in `c_version` and `c_data` per target is probably not correct.  If either change we should just truncate the table.

Reply:
Maybe we need to store it as a global value rather than per target?